ondemand.osc.edu/apps/dashboard/initializers/ood.rb

```diff
@@ -3,14 +3,11 @@ def add_paths
   OodFilesApp.candidate_favorite_paths.tap do |paths|
     # add project space directories
     projects = User.new.groups.map(&:name).grep(/^P./)
-    paths.concat projects.map { |p| Pathname.new("/fs/project/#{p}")  }
-
+    
     # add scratch space directories
     paths << Pathname.new("/fs/scratch/#{User.new.name}")
     paths.concat projects.map { |p| Pathname.new("/fs/scratch/#{p}")  }
 
-    # add ess scratch and project directories
-    paths.concat projects.map { |p| Pathname.new("/fs/ess/scratch/#{p}")  }
     paths.concat projects.map { |p| Pathname.new("/fs/ess/#{p}")  }
```

Comment:
Are you sure these are the right directories? We're removing `/fs/ess/scratch` but keeping `/fs/ess/<project>`?

Seems like we're being told to get rid of all ess and keep regular fs? But this removes one of each.

Reply:
from Summer.

I will suggest to remove all /fs/project/<project-code> and /fs/ess/scratch/<project-code>

Reply:
Yea I'm seeing that's how they want it. Fine by me I guess.